.gitignore

```diff
@@ -4,6 +4,9 @@ target/
 !**/src/main/**
 !**/src/test/**
 
+### Credentials ###
+**/src/main/resources/credentials/*
```

Comment:
remove this. The file is already tracked so need to untrack it first , but it's still better to keep dummy file so leave it as it is :)

Reply:
made those changes. please merge.